terraform/nowcasting/development/main.tf

```diff
@@ -217,29 +222,21 @@ module "nwp-ecmwf" {
   }]
 
   container-env_vars = [
-    { "name" : "AWS_REGION", "value" : var.region },
-    { "name" : "AWS_S3_BUCKET", "value" : module.s3.s3-nwp-bucket.id },
-    { "name" : "ECMWF_AWS_REGION", "value": "eu-west-1" },
-    { "name" : "ECMWF_AWS_S3_BUCKET", "value" : "ocf-ecmwf-production" },
+    { "name" : "MODEL_REPOSITORY", "value" : "ecmwf-realtime" },
+    { "name" : "AWS_REGION", "value" : "eu-west-1" },
+    { "name" : "ECMWF_REALTIME_S3_REGION", "value": "eu-west-1" },
+    { "name" : "ECMWF_REALTIME_S3_BUCKET", "value" : "ocf-ecmwf-production" },
+    { "name" : "ZARRDIR", "value" : format("s3://%s/ecmwf/data", module.s3.s3-nwp-bucket.id) },
     { "name" : "LOGLEVEL", "value" : "DEBUG" },
-    { "name" : "ECMWF_AREA", "value" : "uk" },
     { "name" : "SENTRY_DSN", "value" : var.sentry_dsn },
-    { "name" : "ENVIRONMENT", "value" : local.environment },
```

Comment:
becasue this is the first time, do you mind keepind some of the old one in there, and labelling them legacy, just incase need to roll back. This means can switch between current version and new version easily